tests/transport/epics/ca/test_softioc.py

```diff
@@ -547,9 +549,9 @@ def test_update_datatype(mocker: MockerFixture):
         **record_metadata_from_datatype(attr_w.datatype),
     )
     record_w.set_field.assert_not_called()
-    attr_w.update_datatype(Int(units="m", min=-3))
+    attr_w.update_datatype(Int(units="m", min_alarm=-3))
     record_w.set_field.assert_any_call("EGU", "m")
-    record_w.set_field.assert_any_call("DRVL", -3)
+    record_w.set_field.assert_any_call("LOPR", -3)
```

Comment:
Could we keep this as it was to show that min and DRVL are valid for AttrW, with the AttrR check above validating min_alarm -> LOPR?